build.gradle

```diff
@@ -255,6 +255,8 @@ dependencies {
     compile 'org.apache.commons:commons-math3:3.5'
     compile 'org.apache.commons:commons-collections4:4.1'
     compile 'org.apache.commons:commons-vfs2:2.0'
+    compile 'org.apache.commons:commons-configuration2:2.4'
```

Comment:
Can you migrate to Owner instead of adding a new config library? I've suggested an approach that would allow you to use Owner in this ticket: https://github.com/broadinstitute/gatk/issues/5963

Reply:
See comment below.

No action.